EIPS/eip-6963.md

```diff
@@ -33,9 +33,21 @@ This is achieved by introducing a set of window events to provide a two-way comm
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
 
+### Definitions
+
+Wallet: A user agent that manages keys and facilitates transactions with Ethereum.
+
+Decentralized Application (DApp): A web page that relies upon one or many Web3 platform APIs which are exposed to the web page via the Wallet.
+
+Provider Discovery Library: A library or piece of software that assists a DApp to interact with the Wallet. 
+
 ### Provider Info
 
-Each wallet provider will be announced with the following interface `EIP6963ProviderInfo` that will be used to display to the user:
+Each wallet provider will be announced with the following interface `EIP6963ProviderInfo`. The values in the `EIP6963ProviderInfo` MUST be used as follows:
+
```

Comment:
Note, I dropped the `walletId` property here because UUIDv4 + Wallet Display Name effectively achieves the same goal to uniquely identify the providers I believe. Curious to get some feedback on this change which is the only technical change in this PR at this point. I can split it out into a separate PR as well if it's a bit more controversial.